homeassistant/components/ecoforest/strings.json

```diff
@@ -56,6 +56,11 @@
       "power_level": {
         "name": "Power level"
       }
+    },
+    "switch": {
+      "status": {
+        "name": "Status"
+      }
```

Comment:
```suggestion
```

Reply:
I guess it's related to the previous, status key is likely to be known and no need for translations?

Reply:
well, in this case you dont have <fireplace name> status as switch name but <fireplace name>